src/test/java/duke/command/AddCommandTest.java

```diff
@@ -0,0 +1,108 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AddCommandTest {
+
+
+    // Test to-do Formats
+    @Test
+    public void parseDescription_normalToDo_parsedCorrectly() {
+        try {
+            assertEquals("Make the bed",
+                    ((AddCommand) Parser.parse("todo Make the bed")).parseDescription());
+        } catch (DukeException e) {
+            System.err.println(e);
+        }
+    }
+
+    @Test
+    public void parseDescription_emptyToDo_exceptionThrown() {
+        try {
+            ((AddCommand) Parser.parse("todo")).parseDescription();
+        } catch (DukeException e) {
```

Comment:
You may want to consider using a fail() method. This would ensure that when fail() is reached the test would have failed
```suggestion
 public void parseDescription_emptyToDo_exceptionThrown() {
	        try {
	            ((AddCommand) Parser.parse("todo")).parseDescription();
	            fail() // fail method here which will give an error by Junit if this line is reached. 
	            // This line should not be reached because previous line would throw the exception
```